core/src/main/kotlin/fr/sncf/osrd/stdcm/infra_exploration/InfraExplorer.kt

```diff
@@ -113,8 +114,9 @@ fun initInfraExplorer(
     rawInfra: RawInfra,
     blockInfra: BlockInfra,
     location: PathfindingEdgeLocationId<Block>,
+    temporarySpeedLimitManager: TemporarySpeedLimitManager = TemporarySpeedLimitManager(),
```

Comment:
```suggestion
```

Shouldn't be required there either

Reply:
Done in https://github.com/OpenRailAssociation/osrd/pull/9837/commits/b67a7ef1dfa556d7396fa33ae2a0a38e23bd7aa2